modules/machine-health-checks-pausing.adoc

```diff
@@ -8,7 +8,12 @@
 [id="machine-health-checks-pausing_{context}"]
 = Pausing a MachineHealthCheck resource
 
-During the update process, nodes in the cluster might become temporarily unavailable. In the case of worker nodes, the machine health check might identify such nodes as unhealthy and reboot them. To avoid rebooting such nodes, pause all the `MachineHealthCheck` resources before updating the cluster.
+During the update process, nodes in the cluster might become temporarily unavailable. In the case of worker nodes, the MachineHealthCheck resources might identify such nodes as unhealthy and reboot them. To avoid rebooting such nodes, pause all the `MachineHealthCheck` resources before updating the cluster.
+
+[NOTE]
+====
+As an exception, you do not have to pause the `machine-api-termination-handler` resource, as it does not deploy a new node if the node has been flagged as `notReady`.
```

Comment:
instead of naming a particular MachineHealthCheck, maybe call out the property of the MHC that means you don't need to pause it?  In this case, I expect it's [using the fatal `Terminating` as its `unhealthyConditions`][1], so something like:

> As an exception, you do not have to pause MachineHealthChecks where the `unhealthyConditions` is a fatal condition like `Terminating` being `True`.  Those Machines are toast anyway, so no use waiting before reaping.

or whatever the public-docs analog of that is?

[1]: https://github.com/openshift/machine-api-operator/blob/904c53692e6a98dd03c394b5a66c53dea81fef80/install/0000_30_machine-api-operator_13_machinehealthcheck.yaml#L22-L25

Reply:
Makes sense :) TY!

Reply:
Updated to:
"Some MachineHealthChecks might not need to be paused. If your MachineHealthCheck (MHC) resource has met a fatal condition, it is not necessary to pause the corresponding MHC."